README.md

```diff
@@ -150,28 +150,27 @@ If you wish to build Halide in a separate directory, you can do that like so:
 ### MacOS and Linux
 
 Follow the above instructions to build LLVM or acquire a suitable binary
-release. Then create a separate build folder for Halide and run CMake, pointing
-it to your LLVM installation.
+release. Then change directory to the Halide repository and run:
 
 ```
-% mkdir Halide-build
-% cd Halide-build
-% cmake -DCMAKE_BUILD_TYPE=Release -DLLVM_DIR=/path/to/llvm-install/lib/cmake/llvm /path/to/Halide
-% cmake --build .
+% cmake -DCMAKE_BUILD_TYPE=Release -DLLVM_DIR=$LLVM_ROOT/lib/cmake/llvm -S . -B build
+% cmake --build build
 ```
 
-`LLVM_DIR` should be the folder in the LLVM installation or build tree that
-contains `LLVMConfig.cmake`. It is not required if you have a suitable
-system-wide version installed. If you have multiple system-wide versions
-installed, you can specify the version with `HALIDE_REQUIRE_LLVM_VERSION`. Add
-`-G Ninja` if you prefer to build with the Ninja generator.
+`LLVM_DIR` is the folder in the LLVM installation tree (do not use the build
```

Comment:
Nit: maybe bold the "do not use the build tree by mistake"

Reply:
Good idea. Done.